README.rst

```diff
@@ -0,0 +1,164 @@
+PennyLane-Qulacs Plugin
+######################
+
+.. image:: https://github.com/XanaduAI/pennylane-cirq/blob/master/doc/_static/puzzle_qulacs.png?raw=true
```

Comment:
Do we want to put the same image on the PL-Qulacs repo as well, or should we just use the image from another repo; PL-Cirq in this case (it doesn't really matter that much since it's the same image)?

Reply:
Actually, just realised that it looks for a `puzzle_qulacs.png` image inside of the `pennylane-cirq` repo, which seems a bit weird. Probably should be:
```suggestion
.. image:: https://github.com/XanaduAI/pennylane-qulacs/blob/master/doc/_static/puzzle_qulacs.png?raw=true
```